src/screens/runPlan/PlanItemListItem.tsx

```diff
@@ -21,28 +22,35 @@ export class PlanItemListItem extends React.PureComponent<Props> {
   }
 
   labelTextSize() {
-    switch(this.props.textSize){
+    switch(this.props.student.textSize){
       case 'xl': return styles.labelTextSizeXL;
       case 'l': return styles.labelTextSizeL;
       case 'm': return styles.labelTextSizeM;
       default: return styles.labelTextSizeS;
     }
   }
 
-  markItemPlanAsCompleted = () =>
+  textCase() {
```

Comment:
What do you think about renaming this function to something like 'getPlanItemDisplayName' or so? :)

Reply:
Ok, I'll do it ;)